doc/dune-files.rst

```diff
@@ -1715,25 +1694,25 @@ Examples:
 .. code:: lisp
 
    (dirs *) ;; include all directories
-   (dirs :standard \ ocaml) ;; include all directories except ocaml
-   (dirs :standard \ test* foo*) ;; exclude all directories that start with test or foo
+   (dirs :standard \ ocaml) ;; include all dirs except ocaml
```

Comment:
`ocaml` at the end unless talking about OCaml

Reply:
I think this is the name of a directory so it is OK.